python/tidy/servo_tidy/tidy.py

```diff
@@ -730,6 +734,19 @@ def parse_config(content):
             config[pref] = user_configs[pref]
 
 
+def check_directory_files(directories_for_files):
+    for directory, file_extensions in directories_for_files.items():
+        files = get_file_list(directory, only_changed_files=False, exclude_dirs=[])
```

Comment:
`only_changed_files` and `exclude_dirs` should be brought in as an argument, same as how we do for other functions.